mkshims.ts

```diff
@@ -6,11 +6,9 @@ import {Engine}                     from './sources/Engine';
 import {SupportedPackageManagerSet} from './sources/types';
 
 function shouldGenerateShim(name: string) {
-  if (name === 'vcc.js') {
+  if (name.startsWith(`vendors`))
     return false;
-  } else if (name.startsWith('vendors')) {
-    return false;
-  }
+
   return true;
```

Comment:
nit: Not a fan of the `if(condition) return false else return true` pattern
```suggestion
  return !name.startsWith(`vendors`);
```